Framework/src/TaskRunner.cxx

```diff
@@ -379,11 +382,12 @@ void TaskRunner::publishCycleStats()
   double totalDurationActivity = mTimerTotalDurationActivity.getTime();
 
   // monitoring metrics
-  mCollector->send({ mNumberBlocks, "QC_task_Numberofblocks_in_cycle" });
+  mCollector->send({ mNumberBlocks, "QC_task_Numberofblocks_received_in_cycle" });
```

Comment:
Isn't "blocks" outdated anyway? It might be easier to understand if we use "messages"

Reply:
yes, updating.

Reply:
done